docs/plugins/markdown/revealjs/README.md

```diff
@@ -0,0 +1,155 @@
+# revealjs
+
+<NpmBadge package="@vuepress/plugin-revealjs" />
+
+Add presentation in your VuePress site via Reveal.js.
+
+<!-- more -->
+
+## Usage
+
+```bash
+npm i -D @vuepress/plugin-prismjs@next
+```
+
+```js {7} title=".vuepress/config.js"
+import { revealJsPlugin } from '@vuepress/plugin-revealjs'
+
+export default {
+  plugins: [
+    revealJsPlugin({
+      // Enable reveal.js
+      revealJs: true,
+    }),
+  ],
+}
+```
+
+## Slide Syntax
+
+- Use `---` to split slides
+- Use `--` to split the slides second time (vertical display)
+
+```md
+@slidestart
+
+<!-- slide1 -->
+
+---
+
+<!-- slide2 -->
+
+---
+
+<!-- slide3 -->
+
+@slideend
+```
```

Comment:
When I first browsed the document, I didn't understand the specific usage from the grammar; I only grasped it by looking at the source file of `demo.md` as an example. I suggest adding a concrete example here.